tests/test_model_functions.py

```diff
@@ -62,6 +62,35 @@ def test_graph_for_dae(self):
     #######################################################################
     # this section contains tests for each get_XXX function on Model
 
+    def test_get_derived_quantities(self, basic_model):
```

Comment:
We could also check `test_simple_odes` which does have at least one derived quantity (circle_y_implementation$rhs).

Reply:
ok

Reply:
seems to have 12 (see test) is that what we're expecting?

Reply:
Yes, all the ones it picked up look OK to me - including the two that arise from units-converted connections!